### ISSUE TITLE
Undefined name: ServicePointManager on ]TATIN.UpdateTatin with NETCORE=1

### ISSUE THREAD

**Issue by rikedyp** [aplteam/Tatin#133]
```
      ]TATIN.UpdateTatin
* Command Execution Failed: VALUE ERROR: Undefined name: ServicePointManager
```

```
      ]version
* Assuming "TOOLS.Version"; to see a list of all valid user commands that start with "version", type 
      ]version* -?

 Dyalog  19.0.46960 64-bit Unicode, BuildID be73c399            
 OS      Windows 10 or Windows Server 2016 (10.0.19045) 64-bit  
 Link    3.1.2                                                  
 SALT    2.91                                                   
 UCMD    2.51                                                   
 .NET    .NET 6.0.9                                             
 WS      19.0                                                   
      
```

**Comment by aplteam** [comment]
Which OS is this?

**Comment by mkromberg** [comment]
Windows - note .NET 6, I'll bet the missing name is this guy:
https://learn.microsoft.com/en-us/dotnet/api/system.net.servicepointmanager?view=net-7.0

**Comment by aplteam** [comment]
Fixed in 0.96.2
